id/Human Action/01_03_01.md

```diff
@@ -1,29 +1,29 @@
 
-Many more facts of this kind could be mentioned. But they are no argument against reason, rationalism, and rationality. These dreams have nothing at all to do with the question of whether or not reason is the right and only instrument available for man in his endeavors to attain as much knowledge as is accessible to him. The honest and conscientious truth-seekers have never pretended that reason and scientific research can answer all questions. They were fully aware of the limitations imposed upon the human mind. They cannot be taxed with responsibility for the crudities of the philosophy of Haeckel and the simplism of the various materialist schools.
+Masih banyak lagi fakta seperti ini yang bisa disebutkan. Tapi mereka bukan argumen terhadap alasan, rasionalisme, dan rasionalitas. Mimpi-mimpi ini sama sekali tidak ada hubungannya dengan pertanyaan tentang apakah atau tidak alasan adalah hak dan instrumen yang tersedia bagi manusia dalam upayanya untuk memperoleh pengetahuan sebanyak yang dapat diakses oleh dia. Para pencari kebenaran yang jujur dan teliti tidak pernah berpura-pura bahwa alasan dan penelitian ilmiah dapat menjawab semua pertanyaan. Mereka sepenuhnya menyadari keterbatasan yang dipaksakan pada pikiran manusia. Mereka tidak dapat dikenakan pajak dengan tanggung jawab atas kekasaran filosofi Haeckel dan penyederhanaan berbagai sekolah materialis.
 
-The rationalist philosophers themselves were always intent upon showing the boundaries both of aprioristic theory and of empirical research.[^1] The first representative of British political economy, David Hume, the Utilitarians, and the American Pragmatists are certainly not guilty of having exaggerated the power of man to attain truth. It would be more justifiable to blame the philosophy of the last two hundred years for too much agnosticism and skepticism than for overconfidence in what could be achieved by the human mind.
+Para filsuf rasionalis selalu bertekad untuk menunjukkan batas-batas teori aprioristik dan penelitian empiris. [1] Wakil pertama ekonomi politik dari Inggris, David Hume, Utilitarian, dan Pragmatis Amerika tentu saja tidak bersalah karena telah membesar-besarkan kekuatan manusia untuk mencapai kebenaran. Itu akan lebih dapat dibenarkan untuk menyalahkan filosofi dari dua ratus tahun terakhir ini karena terlalu banyak agnostisisme dan skeptisisme daripada terlalu percaya diri dalam apa yang dapat dicapai oleh pikiran manusia.
```

Comment:
@yhandot could you fix the footnote call? It's missing the ^ (hat)
This is necessary because without it I receive error that completely prevents the book from being rendered.. ^^'